### ISSUE TITLE
Debug tool is too small in phones

### ISSUE THREAD

**Issue by EmmanuelMess** [nicbarker/clay#86]
Testing this library in Android I am having trouble pressing the buttons in the debug panel:
![imagen](https://github.com/user-attachments/assets/83a1c24a-24f4-4ca3-a5b7-1ce9188b0998)

This might be a problem with the font, but please don't make buttons too small.

**Comment by nicbarker** [comment]
You're right, it would certainly be better if there were some customisation options there, including the ability to dock it to the bottom of the screen, rather than the right.

**Comment by EmmanuelMess** [comment]
> You're right, it would certainly be better if there were some customisation options there, including the ability to dock it to the bottom of the screen, rather than the right.

I wouldn't classify it under customization, it would be an explicit minimum size for buttons.
